docs/source/conf.py

```diff
@@ -348,23 +348,42 @@ def _attach_classes_to_module(module, class_data):
 
 
 def _write_resources_rst(class_names, filename, civis_module):
-    with open(filename, 'w') as _out:
-        _out.write('.. _api_resources:\n\nAPI Resources\n=============\n\n')
+    with open(filename, 'w') as resources_rst_file:
+        resources_rst_file.write(
+            ".. _api_resources:\n\n"
+            "API Resources\n"
+            "=============\n\n"
+            ".. toctree::\n"
+            "   :titlesonly:\n\n"
+        )
         for class_name in class_names:
-            name = class_name.title()
-            header = '`{}`\n{}\n'.format(name, '"' * (len(name) + 2))
-            full_path = '.'.join((civis_module, name))
-            _out.write(header)
-            _out.write(_autodoc_fmt.format(full_path, full_path))
+            endpoint_rst_filename_no_ext = f"api_{class_name.lower()}_endpoint"
+            endpoint_rst_filename = f"{endpoint_rst_filename_no_ext}.rst"
+            resources_rst_file.write(f"   {endpoint_rst_filename_no_ext}\n")
```

Comment:
The new auto-generated `api_resources.html` page has just the list of endpoint names, each name pointing to the endpoint's own page.

<img width="934" alt="Screenshot 2024-04-29 at 2 32 33 PM" src="https://github.com/civisanalytics/civis-python/assets/86482098/ffec54c1-7212-4835-9a2b-2bcaed553a82">